ReactWindows/ReactNative.Net46/Views/TextInput/ReactPasswordBoxManager.cs

```diff
@@ -189,7 +191,22 @@ public void SetFontStyle(PasswordBox view, string fontStyleString)
         [ReactProp("placeholder")]
         public void SetPlaceholder(PasswordBox view, string placeholder)
         {
-            throw new NotSupportedException("Placeholder is not supported in WPF.");
+            PlaceholderAdorner.SetText(view, placeholder);
```

Comment:
We are using an adorner layer to paint the placeholder text... This uses attached properties so we call the static "SetText" passing the adorningElement (the PasswordBox) and the text to be set... This will set a property in the adorner that will be painted in the bounds of the PasswordBox when it is rendered.